ontology/yaml/resources/TRANSPORT/entity_types/ELV.yaml

```diff
@@ -20,3 +20,11 @@ ELV_SS:
   - ELV
   - SS  
 
+ELV_EPM_BEA:
```

Comment:
See above comments.  Can use `ELV_SS` or perhaps `ELV_SS_EPM` if floor monitoring is necessary.

Reply:
I can remove the floor monitoring if you prefer, we were under the impression it would be useful for analysis